clients/java/zms/src/main/java/com/yahoo/athenz/zms/ZMSClient.java

```diff
@@ -3594,4 +3594,22 @@ public DomainList getDependentDomainList(String service) {
         }
     }
 
+    /**
+     * Delete expired members from roles and groups.
+     *
+     * @param purgeResources indicates which resource will be purged
+
+     * @throws ZMSClientException in case of failure
+     */
+    public void deleteExpiredMembers(Integer purgeResources) {
```

Comment:
see the comment below for the RDL to optionally return the list of expired members (similar to the support we added for the redux work in UI).